graphql2/graphqlapp/rotation.go

```diff
@@ -413,3 +425,26 @@ func (a *Query) CalcRotationHandoffTimes(ctx context.Context, input *graphql2.Ca
 
 	return result, nil
 }
+
+func getRotationFromISO(start time.Time, dur *timeutil.ISODuration) (rotation.Rotation, error) {
```

Comment:
we should validate that the iso duration only matches one (e.g., if months are specified, days & timepart must be zero and the same for the rest)